Source/Core/Models/Event.cs

```diff
@@ -81,6 +81,7 @@ public static class KnownDataKeys {
             public const string SubmissionMethod = "@submission_method";
             public const string SessionEnd = "sessionend";
             public const string SessionHasError = "haserror";
+            public const string ManualStackingKey = "@manual_stacking_key";
```

Comment:
I like it. I think it's very clear what it does.

If they want to stack on multiple things then they would need to build up their own hash based on those multiple things. I guess it would be nice to allow the value to be a simple string or an object, but I think that just adds more complexity than it's worth.